scripts/build.sh

```diff
@@ -18,3 +22,5 @@ sed -i -e "s/^OS_MIGRATE_VERSION = .*$/OS_MIGRATE_VERSION = '$VERSION'  # update
 ansible-galaxy collection build os_migrate -v --force --output-path releases/
 cd releases
 ln -sf os_migrate-os_migrate-$VERSION.tar.gz os_migrate-os_migrate-latest.tar.gz
+
+python3 -m pip cache purge
```

Comment:
IMO this is not necessary since the above install only happens in a started container, not in the container image itself. But probably doesn't hurt either.